lib/resources/generators/task.js

```diff
@@ -27,7 +27,7 @@ export default class TaskGenerator {
 
   generateSource(functionName) {
     return `import gulp from 'gulp';
-import changed from 'gulp-changed';
```

Comment:
This is a fix that's not actually related to this issue, but what the heck I did it anyway 😅